underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -159,6 +165,26 @@ public SwiftUnderFileSystem(AlluxioURI uri) {
       container.create();
     }
     mContainerPrefix = Constants.HEADER_SWIFT + mContainerName + PATH_SEPARATOR;
+
+    // Assume the Swift user name has 1-1 mapping to Alluxio username.
+    mAccountOwner = Configuration.get(PropertyKey.SWIFT_USER_KEY);
+    short mode = (short) 0;
+    // If there is any container ACL for the Swift user, translates it to Alluxio permission.
+    if (container.getContainerReadPermission().contains(mAccountOwner)
```

Comment:
That's a good point. AFAIK, I intend to use `contains` here because different Swift version and providers might have different ACL format.

E.g. Ceph (Swift)[http://docs.ceph.com/docs/master/radosgw/swift/containerops/#update-a-container-s-acls] uses 
```
POST /{api version}/{account}/{container} HTTP/1.1
Host: {fqdn}
     X-Auth-Token: {auth-token}
     X-Container-Read: *
     X-Container-Write: {uid1}, {uid2}, {uid3}
```
While [swiftstack](https://www.swiftstack.com/docs/cookbooks/swift_usage/container_acl.html#setting-container-read-acl) can have the following:
```
X-Container-Read: .r:*
X-Container-Write: AUTH_bill
```

I'm not sure whether there's a unified way to check a given user's READ/WRITE bit on the container ACL. Therefore, I simply ignore tenant and go with the most straightforward way here,  so that it is guaranteed that the permission is granted as long as there's any `*` or the `username` in the READ_ACL or WRITE_ACL list. 
Please let me know if you have any ideas :-)